tera-templates/macros.html

```diff
@@ -0,0 +1,77 @@
+{# Constructs a title based on the given crate name and version #}
+{% macro doc_title(name, version) %}
```

Comment:
This macro does not appear to be used. Can we remove it?

Reply:
It's used in later stages

Reply:
Please remove it until it is used (by same rationale as above).